core/src/com/mygdx/insanerpg/model/item/stackable/consumable/gem/WeaponGemPrimaryAttribute.java

```diff
@@ -0,0 +1,22 @@
+package com.mygdx.insanerpg.model.item.stackable.consumable.gem;
+
+import java.util.Random;
+
+public enum WeaponGemPrimaryAttribute {
```

Comment:
Are we going to have FlatHp/FlatMp/Regen/etc? because if we do then the minBonus and maxBonus cannot be used nor the getRandomPrimaryAttributeBonus

Reply:
fixed with more min/max constants and used a switch statement in the getRandomPrimaryAttributeBonus

Reply:
also did the same with WeaponGemSecondaryAttributes